docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -153,6 +156,8 @@ result in an access permissions object of:
 }
 ```
 
+_Note:_ The `*` may or may not be proceded by a slash.
```

Comment:
I don't understand the purpose of this note.

Reply:
Several on my team assumed, from seeing the examples, that a `/` always came before a `*`

Seeing this in the morning, I should probably be in the paragraph above...

```diff
- A '`*`' wildcard can be used to replace zero or more characters (this is equivalent to 
- the `.*` regex pattern), and may encompass multiple path segments
+ A '`*`' wildcard can be used to replace zero or more characters (this is equivalent to
+ the `.*` regex pattern). The wildcard may encompass multiple path segments, may be
+ used multiple times, and may appear at any position in the path permission.
```